release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -103,6 +103,24 @@ Warning: short name "ex" could also match lower priority resource examples.test.
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-openshift-sdn-upgrade-block"]
+==== OpenShift SDN network plugin blocks future major upgrades
+
+As part of the {product-title} transition to OVN-Kubernetes as the only supported network plugin, if your cluster uses the OpenShift SDN network plugin, starting with {product-title} 4.16, you cannot upgrade to future major versions of {product-title} without migrating to OVN-Kubernetes. For more information on migrating to OVN-Kubernetes, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
```

Comment:
```suggestion
As part of the {product-title} transition to OVN-Kubernetes as the only supported network plugin, if your cluster uses the OpenShift SDN network plugin, starting with {product-title} 4.16, you cannot upgrade to future major versions of {product-title} without migrating to OVN-Kubernetes. For more information about migrating to OVN-Kubernetes, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
```

Reply:
This makes more sense to me, putting the two key phrases together (if you use... you cannot upgrade)
```suggestion
As part of the {product-title} transition to OVN-Kubernetes as the only supported network plugin, starting with {product-title} 4.16, if your cluster uses the OpenShift SDN network plugin, you cannot upgrade to future major versions of {product-title} without migrating to OVN-Kubernetes. For more information about migrating to OVN-Kubernetes, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
```